README.md

```diff
@@ -1,5 +1,8 @@
+### Billing Report
```

Comment:
- Can you please include the steps used for creating virtual environment ?
- Please mention which version of awscli you have installed and how do you configure aws configs and credentials?
- Briefly mention the installation steps
- Please include Steps or Commands to install the python application ?